docs/source/advanced_install.rst

```diff
@@ -208,7 +208,29 @@ sub-directory of the ahwkit base directory, e.g. by (when being in the base dire
 
     $ export PYTHONPATH=`pwd`/src:$PYTHONPATH
 
-
+CUDA-enabled docker image
+~~~~~~~~~~~~~~~~~~~~~~~~~
+# Get CUDA_ARCH for your GPU from `nvidia-smi`
+
+```bash
+export CUDA_ARCH=$(nvidia-smi --query-gpu=compute_cap --format=csv | sed -n '2 p' | tr -d '.')
+echo $CUDA_ARCH
+```
+
+# Build Image
+
+```console
+docker build \
+--tag aihwkit:cuda \
+--build-arg USERNAME=${USER} \
+--build-arg USERID=(id -u $USER) \
+--build-arg GROUPID=(id -g $USER) \
+--build-arg CUDA_ARCH=${CUDA_ARCH} \
+--build-arg CUDA_VER=11.7 \
+--build-arg UBUNTU_VER=22.04 \
+--build-arg PYTORCH_PIP_URL=https://download.pytorch.org/whl/cu116 \
+--file CUDA.Dockerfile .
+```
```

Comment:
```suggestion
As an alternative to a regular install, a CUDA-enabled docker image can also be
built using the ``CUDA.Dockerfile`` included in the repository.

In order to build the image, first identify the ``CUDA_ARCH`` for your GPU
using from `nvidia-smi` in your local machine::

    export CUDA_ARCH=$(nvidia-smi --query-gpu=compute_cap --format=csv | sed -n '2 p' | tr -d '.')
    echo $CUDA_ARCH

The image can be built via::

    docker build \
    --tag aihwkit:cuda \
    --build-arg USERNAME=${USER} \
    --build-arg USERID=(id -u $USER) \
    --build-arg GROUPID=(id -g $USER) \
    --build-arg CUDA_ARCH=${CUDA_ARCH} \
    --build-arg CUDA_VER=11.7 \
    --build-arg UBUNTU_VER=22.04 \
    --build-arg PYTORCH_PIP_URL=https://download.pytorch.org/whl/cu116 \
    --file CUDA.Dockerfile .

If building your image against a different CUDA or PyTorch version, please
ensure setting the build arguments accordingly.

```

Can you ensure there is a bit more context in the documentation, and using `.rst` syntax? Feel free to accept the suggested change directly!

Reply:
looks good. Sorry I didn't format it well.

Reply:
No worries! `.rst` in particular is quite pesky :+1: